dotnet/src/Connectors/Connectors.AI.OpenAI/FunctionCallBehavior.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Linq;
+using Azure.AI.OpenAI;
+using Microsoft.SemanticKernel.Connectors.AI.OpenAI.AzureSdk;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI;
+
+/// <summary>Represents a behavior for OpenAI function calling.</summary>
+public abstract class FunctionCallBehavior
```

Comment:
Function calling has been deprecated in favour of Tools. We are hoping to get an update to the SDK with the Tools API support before v1.0.

Reply:
Thanks. Presumably that would entail changing the existing APIs, anyway, so introducing these now and then revamping later shouldn't make much of a difference?  When do we expect to pull in a new library that has the tools stuff and is there a plan for adapting?

Reply:
The new library version should be ready next week and we do want to include this for v1.0.